style.css

```diff
@@ -1,595 +1,601 @@
-body {
-  display: flex;
-  align-items: center;
-  flex-direction: column;
-}
-
-.container {
-  display: flex;
-  box-sizing: border-box;
-  height: 652px;
-  width: 375px;
-  border-style: solid;
-  border-width: 0;
-  border-color: black;
-  flex-direction: column;
-}
-
-.toolbar {
-  display: flex;
-  height: 48px;
-  width: 375px;
-  justify-content: space-between;
-}
-
-.logo {
-  margin-left: 12px;
-  margin-top: 4px;
-  margin-bottom: 4px;
-  padding: 10px 12px;
-  width: 100px;
-  height: 40px;
-  display: flex;
```

Comment:
- [ ] **IMPORTANT:** As I mentioned in previous comments, kindly make sure your `Mobile` version still working as expected and not removing its styling, then the styling in the `@media` query will work when you are in a Desktop screen width size.
Example of one behavior of your project: **Header:** **`On mobile`** you will see the `Humbarger/menu` button but not the navigation options(Portfolio, About, Contact) but on **`On Desktop`** it will be the opposite(`Humbarger/menu` button will be hidden and display the navigation options)